corehq/apps/domain/forms.py

```diff
@@ -1485,6 +1490,14 @@ def process_subscription_management(self):
         new_plan_version = DefaultProductPlan.get_default_plan_by_domain(
             self.domain, edition=self.cleaned_data['software_plan_edition'],
         )
+        kwargs = {
+            'auto_generate_credits': True,
+            'date_end': self.cleaned_data['end_date'],
+            'do_not_invoice': False,
+            'internal_change': True,
```

Comment:
@proteusvacuum Should both functions get passed ```internal_change == True```?  What about the other forms on this page?